<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStepPages</class>
 <widget class="QDialog" name="DialogStepPages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_Pages"/>
  </widget>
  <widget class="QPushButton" name="pushButton_Next">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_End">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>终止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
